### ISSUE TITLE
very strange fail

### ISSUE THREAD

**Issue by bubnenkoff** [rejectedsoftware/vibe.d#1911]
s-ludwig, I really can't understand what I am doing wrong. My very small cms based on vibed for dlang.ru worked fine on Windows, but after migration to Debian 8.5 it's failing, and I can't even find the moment of error. Logs are empty. 

I tried to wrap main to try-catch block, and do: `./dcms >> log.txt 2>&1`. All logs are empty. The code is incredible simple. https://github.com/bubnenkoff/dlang.ru

Do you have any ideas how I can detect the reason of issue?

I tried to add `versions "libasync"` to dub.sdl, because I thought that the reason of error is even-loop, but it's seems that error in another place.

Do you have any ideas what I can to try?

**Comment by wilzbach** [comment]
> Do you have any ideas what I can to try?

- What vibe.d version do you use?
- What D compiler?
- Have you tried updating both to the latest?
- Have you tried starting it with gdb to get a stacktrace?

**Comment by bubnenkoff** [comment]
vibed 0.8.1
dmd 2.075.1

I do not know how use gdb with ssh. If you can help I can give connection credentials to you. I do not have enough experience :(

**Comment by s-ludwig** [comment]
I've installed a Debian 8.9 VM and it worked there. I can make an attempt to debug the issue over ssh, you can add my GitHub public key to a local user account:
```
ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAIEAvsTMITpvxaJ9L8JeH9KY9qrRvA6Q0RWVSRmeB+gNI44Mnjdyiok0gC/juSW+a3EFnE7bGLNmSSItF5sMJ9PTQRgQppiyF5mA0z9BXaX4RuD9huHY7rSJ4lXY/rodgmvNbIXZx0FDa5glTkR3VZmlA3eZH3IU5uvIGuupuPd7280= s-ludwig@github.com
```
It's probably too late for today, but I can try to find time tomorrow.

**Comment by bubnenkoff** [comment]
s-ludwig, big thanks for offering help. I decided that reason of failure maybe in slow disk access. I do not know how to check it, but now I optimized code to minimize disk access.

If it would not help, I will ask about help again. Because really error very strange and all was fine on Windows. 

https://stackoverflow.com/questions/46021655/can-be-the-slow-reading-from-the-disk-the-reason-of-exception
